__tests__/src/auth/router/v2.test.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+
+const supertest = require('supertest');
+const { server } = require('../../../../src/server');
+const { db, users } = require('../../../../src/auth/models/index');
+const request = supertest(server);
+
+let adminTest;
+let testUser = {};
+
+beforeAll(async () => {
+  await db.sync();
+  adminTest = await users.create({
+    username: 'triciaS',
+    password: 'passwordy',
+    role: 'admin',
+  });
+});
+
+
+afterAll(async () => {
+  await db.drop();
+});
+
+
+describe('testing v2 routes', () => {
+  test('Bad requests', async () => {
+    const response = await request.post('/api/v2/horse').send({ info: 'bad' });
+    expect(response.status).toEqual(500);
+  });
+
+  // can't get this test passing
+  // test('Cannot create a new food unless your role allows', async () => {
+  //   let response = await request.post('/api/v2/food').send({
+  //     name: 'Birthday cookie',
+  //     calories: 100,
+  //     type: 'Bakery item',
+  //   }).set('Authorization', `Bearer ${testUser.token}`);
+
+  //   expect(response.status).toEqual(500);
+  //   expect(response.body.message).toEqual('Access Denied');
+  // });
```

Comment:
That means that you are sending an empty token here, and the server is sending it's error response from the bearerAuth middleware instead of the acl middleware.